### DISCUSSION TITLE
Simple TV 用家里的无线网看不了，连手机热点可以，求助

### DISCUSSION THREAD

**Discussion by SDH689** [Potato-66/SimpleTV#9] [Q&A] [answered]
Simple TV 用家里的无线网看不了，连手机热点可以，求助。家里是移动的，我手机也是移动

**Comment by Potato-66** [comment, marked as answer]
> Simple TV 用家里的无线网看不了，连手机热点可以，求助。家里是移动的，我手机也是移动

参照REMADE里面第三条，开启ipv6。路由器建议设置定时重启，防止ipv6地址失效
